src/utils/hints/namesHints.js

```diff
@@ -11,4 +11,16 @@ export const namesHints = {
   expired: 'Block height and estimated date and time when the Name was detached from the address.',
   lastBuyer: 'Address of the last account which paid for the Name and the price of the Name.',
   lastOwner: 'Last owner of the Name.',
+  bidder: 'Account of the highest bidder.',
+  bid: 'Price of the highest bid.',
+  ownedSince: 'Block height and estimated date and time when the account became the owner of the Name.',
+  status: 'Current state of the Name in the blockchain.',
+  customPointers: 'User-defined pointers that point to any valid value.',
+  accountPointer: 'Account that the Name points to.',
+  channelPointer: 'State Channel that the Name points to.',
+  contractPointer: 'Smart Contract that the Name points to.',
+  oraclePointer: 'Oracle that the Name points to.',
+  eventTransactionHash: 'Transaction hash of the Name event.',
+  eventTime: 'Block height and estimated date and time when the event was triggered.',
+  activity: 'Activity explains what type of event was triggered.',
```

Comment:
Maybe simply `Type of event that was triggered.`, the first part sounds unnecessary.